### ISSUE TITLE
Observe not working

### ISSUE THREAD

**Issue by blind76** [KartikTalwar/gmail.js#275]
Hi

I'm using gmail.js in chrome extension and most of observers not working. 'view_thread' works but other doesn't. I need to trig some function when inbox is open so I tried few observers () and every time I get "    //if api.dom.email called with invalid element/id"

**Comment by YesThatAllen** [comment]
We're running into a similar issue when we're trying to reference the `From:` in an email.

```
gmail.js:567 api.dom.email called with invalid element/idGmail.api.tools.error @ gmail.js:567Gmail.api.dom.email @ gmail.js:2337(anonymous function) @ VM18710:54
VM18710:79 Sending post for undefined
```

![screenshot_1795](https://cloud.githubusercontent.com/assets/2879972/16710724/ee84b6f4-45fe-11e6-9f7e-2f51264ec3ed.png)

**Comment by josteink** [comment]
From what I can tell, this bug has been fixed by PR #256, so I'm closing this one.

Please re-open if you still have something you need resolved.
